components/bear-market-airdrop/Blocks/FAQ.tsx

```diff
@@ -0,0 +1,239 @@
+/* eslint-disable no-restricted-imports */
```

Comment:
I used the chakra variants since I had already styled them to the designs, didn't want to have the pr be blocked because I would have to adapt the tw-components to be styled the way I wanted it to be.

Will fix those tomorrow.